<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>871</width>
      <height>900</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>501</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>501</width>
       <height>583</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Задание ключевых кадров</string>
     </attribute>
     <widget class="QPushButton" name="btnAdd">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить новый ключевой кадр</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbFrames">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>361</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Список ключевых кадров</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnView">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Просмотреть все ключевые кадры</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="btnLast">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавлять ключевые кадры в конец</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="btnCurrent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>501</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавлять ключевые кадры после текущего</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeleteAll">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>500</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить все ключевые кадры</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeleteCurr">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>410</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить текущий кадр</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>501</width>
       <height>583</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Анимация</string>
     </attribute>
     <widget class="QPushButton" name="btnPlay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Запустить анимацию</string>
      </property>
     </widget>
     <widget class="QSlider" name="Speed">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Частота кадров</string>
      </property>
     </widget>
     <widget class="QSlider" name="Duration">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Продолжительность анимации</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDuration">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>3 сек</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblFrameRate">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1 кадр/сек</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStop">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Остановить анимацию</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSafe">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>430</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить анимацию</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnShowExample">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>510</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Запустить пример</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtAnimName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>421</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>340</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Имя анимации</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>423</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите имя файла для загрузки модели</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtInput">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInput">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>181</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отобразить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chSkelet">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Скелет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chBody">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Оболочку</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Помощь</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1420</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
